example/vpc-and-vpn/main.tf

```diff
@@ -0,0 +1,12 @@
+module "vpc-and-vpn" {
+    source = "../modules/vpc-and-vpn"
```

Comment:
should this be `../../modules/vpc-and-vpn`?

Reply:
/ping @nikever

Reply:
✅